### ISSUE TITLE
Implement DwC-A import

### ISSUE THREAD

**Issue by informatics-dev** [NaturalHistoryMuseum/scratchpads2#5294]
[GIASIPartnership](http://giasipartnership.myspecies.info)

 - Original ID: 4228
 - Added by **Simon Rycroft**
 - Scratchpads URL: 

### Description:
The GIASIP website will acquire content from an instance of the IPT.  Data will be provided in a DwC-A format, which means we need to be able to read this format, and import all content from it.  I would imagine the DwC-A, and specifically the meta.xml file could be read, and different Feeds importers created based on the structure.

**Comment by Archilegt** [comment]
No, this is a different technical challenge. It should not be undertaken by developers alone. We need to work together with biologists using the DwC-A and implement a dashboard to notify of and resolve data harmonization problems that will definitely appear during import.
